cmake/external/dnnl.cmake

```diff
@@ -1,72 +1,22 @@
 include (ExternalProject)
 
+
 set(DNNL_URL https://github.com/oneapi-src/onednn)
 # If DNNL_TAG is updated, check if MKLML_VERSION and platform.cmake.patch need to be updated.
 set(DNNL_TAG v1.7)
-set(MKLML_VERSION 2019.0.5.20190502)
+
 
 if(WIN32)
-  set(MKLML_OS_VERSION_STR "win")
-  set(MKLML_FILE_EXTENSION "zip")
   set(DNNL_SHARED_LIB dnnl.dll)
-  set(DNNL_IMPORT_LIB dnnl.lib)
-  if(onnxruntime_USE_MKLML)
-    # Windows-only updated MKLML binary which contains fix for thread cleanup hang.
-    set(MKLML_VERSION 2020.0.20190813)
-    set(MKLML_SHARED_LIB mklml.dll)
-    set(MKLML_IMPORT_LIB mklml.lib)
-    set(IOMP5MD_SHARED_LIB libiomp5md.dll)
-    set(IOMP5MD_IMPORT_LIB libiomp5md.lib)
-  endif()
+  set(DNNL_IMPORT_LIB dnnl.lib)  
 else()
-  set(MKLML_FILE_EXTENSION "tgz")
   if (APPLE)
     set(DNNL_SHARED_LIB libdnnl.1.dylib)
-    set(MKLML_OS_VERSION_STR "mac")
-    if(onnxruntime_USE_MKLML)
-      set(MKLML_SHARED_LIB libmklml.dylib)
-      set(IOMP5MD_SHARED_LIB libiomp5.dylib)
-    endif()
   else()
     set(DNNL_SHARED_LIB libdnnl.so.1)
-    set(MKLML_OS_VERSION_STR "lnx")
-    if(onnxruntime_USE_MKLML)
-      if(onnxruntime_USE_OPENMP)
-        set(MKLML_SHARED_LIB libmklml_gnu.so)
-      else()
-        set(MKLML_SHARED_LIB libmklml_intel.so)
-        set(IOMP5MD_SHARED_LIB libiomp5.so)
-      endif()
-    endif()
   endif()  
 endif()
 
-if (onnxruntime_USE_MKLML)
-  set(MKLDNN_VERSION_SHORT v0.20)
-  set(MKLML_URL https://github.com/intel/mkl-dnn/releases/download/${MKLDNN_VERSION_SHORT}/mklml_${MKLML_OS_VERSION_STR}_${MKLML_VERSION}.${MKLML_FILE_EXTENSION})
-
-  ExternalProject_Add(project_mklml
-    PREFIX mklml
-    URL ${MKLML_URL}
-    CONFIGURE_COMMAND ""
-    BUILD_COMMAND ""
-    UPDATE_COMMAND ""
-    INSTALL_COMMAND ""  )
-
-  set(MKML_DIR ${CMAKE_CURRENT_BINARY_DIR}/mklml/src/project_mklml)
-  set(MKLML_INCLUDE_DIR "${MKML_DIR}/include")
-  set(MKLML_LIB_DIR "${MKML_DIR}/lib")
-  link_directories(${MKLML_LIB_DIR})
-endif()
-
-if (onnxruntime_USE_DNNL AND onnxruntime_DNNL_GPU_RUNTIME STREQUAL "ocl" AND onnxruntime_DNNL_OPENCL_ROOT STREQUAL "")
```

Comment:
this section shouldn't be removed right? it's related to dnnl training which was a recent contribution.

Reply:
Oh!! That's a mistake. Sorry.